internal/host/transition_test.go

```diff
@@ -1028,11 +1029,11 @@ var _ = Describe("Refresh Host", func() {
 			models.HostStageConfiguring,
 			models.HostStageWaitingForIgnition,
 			models.HostStageInstalling,
-			"not_mentioned_stage",
+			fakeStage,
```

Comment:
It looks like it describes an invalid stage, "fakeStage" seems a bit confusing. Maybe "invalidStage" or "unexpectedStage"?